taskRequests/README.md

```diff
@@ -0,0 +1,15 @@
+# Task Request
+
+Firestore Task Requests collection data model
+
+```
+id: <id>,
+taskId: <id>,
+title: string,
+requestedBy:  [string],
+purpose: string,
+type: string,
```

Comment:
Is it possible to have a set of types for task type?
If yes, then use them here.